### PR TITLE
NIFI-4098 - Adjust ParseCEF to re-use bean validator in order to impr…

### PR DESCRIPTION
**Pull request by trixpan**
…ove performance

Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [X] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [X] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [X] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [X] Is your initial contribution a single, squashed commit?

### For code changes:
- [X] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [X] Have you written or updated unit tests to verify your changes?
- [X] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [X] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [X] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [X] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [X] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by pvillard31** [comment]
From a coding point of view, it LGTM. On the L&N side, I'd appreciate another pair of eyes to be sure we're good to go. @joewitt ?

Also, @trixpan, I think the updated dependency is now available in Maven central, could you force another Travis build?

File: nifi-assembly/NOTICE [hunk 1]
```diff
@@ -44,6 +44,20 @@ Apache Software License v2
 
 The following binary components are provided under the Apache Software License v2
 
+  (ASLv2) Apache BVal Project
+    The following NOTICE information applies:
+      Apache BVal project
+     Copyright 2010-2016 The Apache Software Foundation.
+
+     This product includes software developed by
```
Comment (joewitt) [inline nifi-assembly/NOTICE]: please remove lines 52-56.

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-nar/src/main/resources/META-INF/NOTICE [hunk 2]
```diff
@@ -26,6 +26,20 @@ Apache Software License v2
 
 The following binary components are provided under the Apache Software License v2
 
+  (ASLv2) Apache BVal Project
+    The following NOTICE information applies:
+      Apache BVal project
+     Copyright 2010-2016 The Apache Software Foundation.
+
+     This product includes software developed by
```
Comment (joewitt) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-nar/src/main/resources/META-INF/NOTICE]: please remove lines 34-38.

**Comment by joewitt** [comment]
have verified L&N.  Fixed an unrelated L&N finding and cleaned up the things i left comments on.  Checked all dependencies and they're covered too.  Good job.  +1 merging to master

**Pull request closed without merging**